.goreleaser.yaml

```
@@ -0,0 +1,108 @@
+version: 1
+
+dist: ./build
+
+before:
+  hooks:
+    - go mod tidy
+    - make deps:playground
+
+after:
+  hooks:
+    - cmd: docker pull {{ .Env.GITHUB_REPOSITORY }}:latest
+    - cmd: docker run --rm {{ .Env.GITHUB_REPOSITORY }}:latest
+
+builds:
+  - id: "defradb"
+    main: ./cmd/defradb
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+
+  - id: "defradb+pg"
+    main: ./cmd/defradb
+    flags:
+      - -tags=playground
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+
+
+archives:
+  - id: defradb+pg
+    builds: 
+      - defradb+pg
```

todo: Why not underscore _ instead of + plus haha?

```suggestion
  - id: defradb_pg
    builds: 
      - defradb_pg
```

---

✅